packages/zowe-explorer/src/uss/utils.ts

```diff
@@ -75,7 +75,7 @@ export function disposeClipboardContents(): void {
 }
 
 export async function autoDetectEncoding(node: IZoweUSSTreeNode, profile?: imperative.IProfileLoaded): Promise<void> {
-    if (node.binary || node.encoding !== undefined) {
+    if (node == null || node.binary || node.encoding !== undefined) {
```

Comment:
Do we have specific behavior defined for when `node.encoding === null`?

Reply:
I believe @traeok is handling this in the FileSystemProvider branch - instead of `binary: boolean` and `encoding: string`, we can pass the `ZosEncoding` object around for all encoding related operations 😋